docs/src/index.js

```diff
@@ -134,6 +134,7 @@ const Page = props => {
     <>
       <Helmet>
         <title>{title}</title>
+        <link rel='shortcut icon' type='image/x-icon' href='/favicon.ico'/>
```

Comment:
Thanks for looking into this! Whatever you end up using to generate the icon with, can you please add an `npx` script to the `package.json`? – see how repng is used for the other images

Reply:
I use [https://realfavicongenerator.net/](url) link to generate favicon from static/logo.svg file

Reply:
Thanks! If there’s not an installable package this will likely fall out of sync over time

Reply:
As of my concern, there is no installable package which likely fulfills our requirements

Reply:
Maybe one of the following npm packages? I just did a quick search for "svg to ico npm" on Google:

- https://www.npmjs.com/package/favicons
- https://www.npmjs.com/package/icon-gen

Reply:
It could also be generated from the `png` output from repng, if that makes finding a library easier

Reply:
Is there any reason *not* to use a `.png` file for the favicon? I assume most modern browsers support this now, but haven't done any research